### PR TITLE
[1LP][RFR][NOTEST] Add BZ links to the test cases

### PR DESCRIPTION
**Pull request by jaryn**
Add BZ links to covering cases

### CODE REVIEW

File: cfme/tests/configure/test_register_appliance.py [hunk 1]
```diff
@@ -196,6 +196,8 @@ def test_rh_registration(appliance, request, reg_method, reg_data, proxy_url, pr
 def test_rhsm_registration_check_repo_names(
         temp_appliance_preconfig_funcscope, soft_assert, appliance):
     """ Checks default rpm repos on a fresh appliance
+    Bugzilla:
+        1694073
```
Comment (mshriver) [inline cfme/tests/configure/test_register_appliance.py]: Please put them in coverage/automates meta markers if that applies. This is a nice way to list BZ's that are somehow related to the bug, maybe the original RFE, but if there is specific coverage for these, marking them is more useful for tooling around automatically determining our coverage.

Comment (jaryn) [reply cfme/tests/configure/test_register_appliance.py]: done

Comment (john-dupuy) [reply cfme/tests/configure/test_register_appliance.py]: No need to remove them in the future, but they are not necessary like the markers are.

**Comment by john-dupuy** [comment]
Not really blockers-only, but it's bz related.

**Pull request merged**